src/opossum_lib/file_generation.py

```diff
@@ -38,8 +39,8 @@
 def write_dict_to_file(
     opossum_information: OpossumInformation, file_path: Path
 ) -> None:
-    with file_path.open("w") as out:
-        json.dump(to_dict(opossum_information), out, indent=4)
+    with ZipFile(file_path, "w", compression=ZIP_DEFLATED, compresslevel=5) as z:
```

Comment:
I'd suggest to extract the magic number 5 into a constant

Reply:
I rebased the branch on main to extract and add the constant to the newly added `constants` module (in #44 merged) and force-pushed.